src/Microsoft.PowerShell.ConsoleHost/resources/ConsoleHostUserInterfaceSecurityResources.resx

```diff
@@ -123,4 +123,10 @@
   <data name="PromptForCredential_Password" xml:space="preserve">
     <value>Password for user {0}: </value>
   </data>
+  <data name="PromptForCredential_ConfirmPassword" xml:space="preserve">
+    <value>Confirm password for user {0}: </value>
+  </data>
+    <data name="PasswordMismatch" xml:space="preserve">
+    <value>Passwords do not match. Enter your credentials. </value>
```

Comment:
I am nor sure that I like the message but at least we should remove extra space.
```suggestion
    <value>Passwords do not match. Enter your credentials.</value>
```